engine/engine.go

```diff
@@ -317,6 +317,13 @@ func (p *Engine) ReceiveSubject(ctx context.Context, providerID, subjectID, call
 		newStatus = alert.StatusActive
 	case notification.ResultResolve:
 		newStatus = alert.StatusClosed
+	case notification.ResultEscalate:
+		newStatus = alert.StatusTriggered
+		alertIds, err := p.a.EscalateMany(ctx, []int{cb.AlertID})
+		if err != nil {
+			log.Logf(ctx, "Error while escalating "+err.Error())
+		}
+		cb.AlertID = alertIds[0]
```

Comment:
We actually don't want to do any status updates here, and just leave it at escalation

```suggestion
		_, err := p.a.EscalateMany(ctx, []int{cb.AlertID})
		if err != nil {
			return fmt.Errorf("escalate alert: %w", err)
		}
		return nil
```

Reply:
Got it, updated the code. Thanks.